create-product-service/handler.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+const AWS = require("aws-sdk");
+const { v4: uuidv4 } = require("uuid");
+const dynamodb = new AWS.DynamoDB.DocumentClient();
+
+module.exports.createProduct = async (event) => {
+    const product = JSON.parse(event.body);
+    const id = uuidv4();
+    const productsParams = {
+        TableName: "products",
+        Item: {
+            title: product.title,
+            description: product.description,
+            price: product.price,
+            id
+        }
+    };
+
+    const stocksParams = {
+        TableName: "stocks",
+        Item: {
+            product_id: id,
+            count: product.count || 1
+        }
+    };
+
+    try {
+        await dynamodb.put(productsParams).promise();
+        await dynamodb.put(stocksParams).promise();
+        return {
+            statusCode: 200,
+            body: JSON.stringify({ message: "Product created successfully" })
+        };
+    } catch (error) {
+        console.error(error);
+        return {
+            statusCode: 400,
```

Comment:
I think 500 code is more suitable here, since the code inside try block depends on DynamoDb service availability more than user input correctness.

Reply:
I agree with you, but then I read the task and corrected that according to the requirements 

<img width="938" alt="image" src="https://user-images.githubusercontent.com/36797396/224963400-e71d5a56-050d-490b-aa10-1105b991f3f3.png">

Reply:
fixed